CHANGELOG.md

```diff
@@ -1,3 +1,102 @@
+# v1.20.0 (2024-05-13)
+
+## OS Changes
+* Update third party packages ([#3939])
+* Enable file system encryption in 5.15 and 6.1 kernels ([#3906], [#3908])
+* Backport fix for loading SELinux modules ([#3907])
+* Add Fabric Manager support ([#3873])
+* Update host containers ([#3947])
+* Add setting to configure ntp options ([#3852] thanks @domgoodwin)
+* Include swap utilities ([#3829])
+* Update kernels to 6.1.87, 5.15.156, 5.10.215 ([#3934], [#3930])
+
+## Orchestrator Changes
+
+### Kubernetes
+* Add Kubernetes 1.30 variants ([#3859], [#3936])
+* Add container-runtime settings to `aws-k8s-*-nvidia` variants ([#3945])
+* Drop Kubernetes 1.25 Metal and VMware variants ([#3896])
+
+### ECS
+* Update ecs-agent to 1.82.3 ([#3939])
+* Use systemd drop-ins to configure the ECS agent ([#3834])
+
+## Build Changes
+* Update twoliter and the SDK ([#3938], [#3880], [#3885], [#3900])
+* Remove libbzip2 ([#3944])
+* Pessimize Rust builds that require the AWS SDK ([#3892])
+* Reduce variant matrix in CI/CD ([#3863])
+* Document package build tools for go dependencies ([#3882])
+* Update Go lints in CI/CD ([#3882])
+* Out-of-tree build enablement
+  * systemd: use build defaults and kernel parameters for unified cgroups ([#3886], [#3935])
+  * early-boot-config: Use standalone provider binaries to fetch user data ([#3637], [#3890])
+  * logdog: retrieve settings via API client ([#3946])
+  * netdog: remove conditional compilation, add hostname helpers ([#3700], [#3898])
+  * schnauzer: add if_not_null template helper ([#3715])
+  * static-pods: remove conditional compilation, switch to config file ([#3891], [#3927], [#3913])
+  * host-containers: switch to config file ([#3777], [#3842])
+  * bootstrap-containers: switch to config file ([#3724])
+  * corndog: switch to config file ([#3715])
+  * prairiedog: switch to config file ([#3713], [#3814], [#3836])
+  * thar-be-updates: switch to config file ([#3721])
+  * updog: use modeled types ([#3901])
+  * kernel: remove variant sensitivity ([#3897], [#3905], [#3932])
+* FIPS enablement
+  * add FIPS report to the API ([#3894])
+  * add release-fips package for FIPS functionality ([#3893])
+  * build Go binaries for FIPS and non-FIPS ([#3887])
```

Comment:
Would be good to verify (via https://github.com/joeyespo/grip or similar) that this is the right syntax for nested bullet lists (I tend to mess this up)

Reply:
I verified this by previewing the file in GitHub. My eyeballs report: yes, indented bullet list.

Reply:
vscode has a nifty shorcut `<super>+shift+v` to preview which I used.

Reply:
>  I verified this by previewing the file in GitHub. My eyeballs report: yes, indented bullet list.

TIL! https://github.com/bottlerocket-os/bottlerocket/blob/3152f474fa15ca7c93cf8421a9c533ea9b905f56/CHANGELOG.md